src/lib/message/L1Transaction.ts

```diff
@@ -196,12 +245,24 @@ export class L1TransactionReceipt implements TransactionReceipt {
   ): Promise<L1ToL2MessageReaderOrWriter<T>[]>
   public async getL1ToL2Messages<T extends SignerOrProvider>(
     l2SignerOrProvider: T
-  ): Promise<L1ToL2MessageReader[] | L1ToL2MessageWriter[]> {
+  ): Promise<
+    L1ToL2MessageReader[] | L1ToL2MessageWriter[] | ClassicL1ToL2MessageReader
```

Comment:
My concern about adding another type here is potential typescript errors on FE. No errors in SDK related to this.

Reply:
Added tests, now trying to make it better so that Classic type is not returned together with nitro types, could cause some errors. 

My thinking is maybe we should structure classic data to have similar output to nitro?